pkg/renovate/bump/bump.go

```diff
@@ -140,7 +140,8 @@ func New(ctx context.Context, opts ...Option) renovate.Renovator {
 			RecurseNodes().
 			Filter(yit.WithMapValue("git-checkout"))
 
-		for gitCheckoutNode, ok := it(); ok; gitCheckoutNode, ok = it() {
+		// Only run updateGitCheckout once for the first git-checkout node
```

Comment:
This sounds like a limitation. I'd consider instead to request support for checking updates for any of the git-checkouts.